scripts/infra

```diff
@@ -61,13 +61,15 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
                 terraform remote push
                 ;;
             plan)
+                echo "Updating AWS Batch job definitions"
                 pushd "../aws-batch"
                 # TODO: Only trigger this if the pfb-analysis container has changed, otherwise skip
                 #       and reuse old version
                 # TODO: This is a bit awkward. If we `plan` but never `apply`, we'll have created
                 #       an orphaned job definition that never gets used.
-                echo "Updating AWS Batch job definitions"
-                BATCH_JOB_NAME_REVISION=$(python update_job_definition.py pfb-analysis-run-job.json "${PFB_AWS_ECR_ENDPOINT}/pfb-analysis:${GIT_COMMIT}")
+                docker-compose build
```

Comment:
That makes a lot of sense. Since this is (hopefully) temporary, I'm fine with leaving things as they are.